flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/api/PlannerConfigOptions.java

```diff
@@ -30,7 +31,8 @@
 	// ------------------------------------------------------------------------
 	//  Optimizer Options
 	// ------------------------------------------------------------------------
-
+	@Documentation.ExcludeFromDocumentation
```

Comment:
please specify reasons for why config options are excluded from the documentation. This applies to all options that are excluded, not just this one.

Reply:
General users are not easy to understand and have no need to use these configs in most scenes.  Advanced users can tune performance of some special cases according to them.
So I do not want to place these professional configs to the documentation.

Reply:
document the reason accordingly

Reply:
Added the reason.